views/show.tmpl

```diff
@@ -112,27 +112,102 @@
+            {{if ne .SeasonNum 0}}
+              <tr class="timeslot-filter-season" data-SeasonID="{{.SeasonID}}">
+                <td>{{.SeasonID}}</td>
+                <td style="width:10px">{{.SeasonNum}}.</td>
+                <td>{{.FirstTime.Format "Jan 2006"}}</td>
+                <td style="width:10px"><span class="badge badge-pill badge-light">
+                  {{.NumEpisodes.Value}}</span>
+                </td>
+              </tr>
+            {{end}}
+          {{end}}
+          </tbody>
+        </table>
+      </div>
+      <div id="timeslot-latest" class="col-12 col-md-6 col-xl-4 p-4 bg-white">
+        <h3>Latest Episode</h3>
+        {{if ne .PageData.LatestTimeslot.TimeslotID 0}}
+        <p>Catchup with the latest episode from <strong>{{.PageData.LatestTimeslot.StartTime.Format "Monday, _2 Jan 2006"}}</strong></p>
+        {{if .PageData.LatestMixcloud}}
+          <iframe width="100%" height="120" src="https://www.mixcloud.com/widget/iframe/?feed={{.PageData.LatestTimeslot.MixcloudStatus}}&hide_cover=1&light=1"
+            frameborder="0"></iframe>
+        {{else}}
+          <p>
+            <strong>This episode isn't available on {{.PageContext.ShortName}} {{.PageContext.ODName}}, sorry about that.</strong>
+          </p>
+        {{end}}
+        <a href="/schedule/shows/timeslots/{{.PageData.LatestTimeslot.TimeslotID}}/">View full episode information...</a>
+        {{else}}
+          <p>
+            <strong>No episodes of this show have been broadcast yet.</strong>
+          </p>
         {{end}}
-        </ul>
+      </div>
+{{with .PageData}}
+      <div id="timeslot-list" class="col-12 col-md-6 col-xl-4 bg-off-white">
+        <h2 class="inline">Episodes</h2><h3 id="timeslot-season-title" class="inline"></h3>
+        <table id="timeslots" class="table timeslots-table">
+          <thead>
+            <tr>
+              <th>SeasonID</th>
+              <th>Season</th>
+              <th>Episode</th>
+              <th>Broadcast Time</th>
+            </tr>
+          </thead>
+          <tbody>
+          {{range .Timeslots}}
+            {{if ne .TimeslotNum 0}}
+              <tr>
+                <td>{{.Season.SeasonID}}</td>
+                <td>{{.Season.SeasonNum}}</td>
+                <td style="width:10px">{{.TimeslotNum}}.</td>
+                <td><a href="/schedule/shows/timeslots/{{.TimeslotID}}/">{{.StartTime.Format "Monday, _2 Jan 2006 at 15:04"}}</a></td>
+              </tr>
+            {{end}}
+          {{end}}
+          </tbody>
+        </table>
       </div>
     </div>
   </div>
-</div>
+</div> <!-- /outer containter -->
 {{end}}
 {{end}}
 {{define "footer-scripts"}}
-  <script src="{{url "/js/jquery.prettySocial.min.js"}}"></script>
-  <script type="text/javascript" class="source">
-    $(".prettySocial").prettySocial();
-  </script>
+<link rel="stylesheet" type="text/css" href="https://cdn.datatables.net/1.10.18/css/dataTables.bootstrap4.min.css">
```

Comment:
hardcoded https (i know it doesn't matter)

though, why are these using a CDN, when jquery is local? bit inconsistent

Reply:
Hmm, jQuery was CDN as well, last time I checked.